src/main/java/appeng/block/misc/BlockSkyCompass.java

```diff
@@ -164,4 +186,31 @@ public void addCollidingBlockToList( final World w, final BlockPos pos, final Ax
 	{
 
 	}
+
+	@Override
+	public EnumBlockRenderType getRenderType( IBlockState state )
+	{
+		return EnumBlockRenderType.ENTITYBLOCK_ANIMATED;
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public TileEntitySpecialRenderer<? extends AEBaseTile> getTESR()
+	{
+		return new SkyCompassTESR();
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public ItemMeshDefinition getItemMeshDefinition()
+	{
+		return is -> new ModelResourceLocation( "appliedenergistics2:tile.BlockSkyCompass", "normal" );
```

Comment:
This is Java8 synax. Is there any final decision about using Java8 made? If so we should use more of it. For example replacing the guava `Optional` with java8 `Optional`